src/app/portlets/dot-edit-content/services/dot-edit-content-html.service.ts

```diff
@@ -58,13 +56,50 @@ export class DotEditContentHtmlService {
         });
     }
 
+    /**
+     * Remove a contentlet from the DOM by inode and update the page model
+     *
+     * @param {string} inode
+     * @memberof DotEditContentHtmlService
+     */
     removeContentlet(inode: string): void {
         const doc = this.getEditPageDocument();
         const contenletEl = doc.querySelector(`div[data-dot-inode="${inode}"]`);
         contenletEl.remove();
         this.pageModelChange.next(this.getContentModel());
     }
 
+    /**
+     * Render contentlet in the DOM after edition.
+     *
+     * @param {*} contentlet
+     * @memberof DotEditContentHtmlService
+     */
+    renderEditedContentlet(contentlet: any): void {
```

Comment:
we need a new test for this method

Reply:
Yes... same thing... we don't have tests for any of this methods, refactor will do.